backend/public/setup.php

```diff
@@ -240,8 +242,6 @@
     }
 }
 
-$conn->commit();
```

Comment:
Das hab ich wahrscheinlich mit #571 kaputt gemacht. Vorher wurde beim Bootstrappen eine transaction eröffnet. Wäre eine alternative manuell eine transaction zu eröffnen + try/catch/rollback? Dann hätten wir keine halben Daten im System. Oder wird sowieso automatisch alles in eine Transaction gekapselt?